docs/source/appendices/components.rst

```diff
@@ -0,0 +1,119 @@
+Components
+==========
+
+
+On an operational level, the seven components of the BDSS methodology outlined
+in the INTRODUCTION can be described at a high-level as follows:
+
+
+A decision-making strategy
+--------------------------
+The BDSS supports an organization to take decisions for development/maintenance of
+a product and/or production process.  Information transactions and decisions are
+connected by a ‘workflow’ which combines individual actions for information requests,
+information input, decision-making, and decision authorization for a specific
+decision-making process. Product and/or manufacturing process decisions are driven
+by KPIs (Key Performance Indicators), their targets and respective acceptance
+criteria or constraints. KPIs may be business (financial, marketing, regulations,
+resource availability, etc.) and/or technical (material or product properties,
+manufacturability, etc.), or a combination of the two.
+
+The Force platform will be open and based on standards
+such that all components (modelling, MCOs, KPIs, dataspaces, etc.) can seamlessly
+exchange relevant information allowing an optimal business decision workflow to be
+developed rapidly while the availability of cognitive elements in the user interfaces
+will allow use even by non-experts in the materials modelling. Data from modelling
+will be augmented with existing data including chemical and mechanical properties,
+process characteristics, commercial and business data. In particular, information
+on costs of raw and compound materials, processing, market trends and customer needs
+and demands will be included. Decisions based on this variety of information will be
+facilitated by dashboards and multi-criteria-optimisation (MCO) tools. These tools
+are particularly useful to identify best compromises when dealing with conflicting
+objectives, which  are typical in materials design (e.g. lighter-stiffer, cheap-high
+purity, etc.).
+
+Expertise
+---------
+The BDSS end-users are divided into three high-level personas - Business Manager,
+Senior Manager, and Operations Personnel.
+
+Computational Tools
+-------------------
+The RoMM provides a classification of models and tools developed to support
+materials modelling. Market-wise, the majority of materials discrete modelling
+code is developed and maintained by SMEs, and it is well established that there
+are many more codes reaching a high quality within academia/research as evidenced
+in the RoMM.
+
+Continuum modelling tools (openFoam), atomistic modelling
+tools (Gromacs), simulation platform (Simphony with the FORCE BDSS WORKFLOW Manager),
+MCO tools (Dakota,…), data-driven modelling tools (IBM Watson).
```

Comment:
This section is missing a lot of context, so it feels out of place. We probably don't need to include it for a tutorial document